app/views/templates/payments/WhatYouOweChargeHelper.scala

```diff
@@ -30,21 +30,30 @@ class WhatYouOweChargeHelper @Inject()(payment: OpenPaymentsModel, hasDirectDebi
     case `vatReturnDebitCharge` =>
       s"${messages.apply("openPayments.forPeriod")} ${displayDateRange(payment.start, payment.end)}"
     case `officerAssessmentDebitCharge` => messages.apply("openPayments.officersAssessment")
-  }
+    case `vatDefaultSurcharge` =>
+      s"${messages.apply("openPayments.surcharge", {displayDateRange(payment.start, payment.end)})}"
+    case `vatCentralAssessment` =>
+      s"${messages.apply("openPayments.centralAssessment", {displayDateRange(payment.start, payment.end)}).trim}${messages("openPayments.centralAssessmentSubmit")}"}
```

Comment:
Could you split this into two lines to suppress file length IntelliJ warning please.